src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -165,10 +160,6 @@ export const ImportAccounts = ({ chainId, onClose }: ImportAccountsProps) => {
     () => wallet instanceof MetaMaskShapeShiftMultiChainHDWallet,
     [wallet],
   )
-  const highestAccountNumberForChainIdFilter = useMemo(() => ({ chainId }), [chainId])
-  const highestAccountNumber = useAppSelector(state =>
-    selectHighestAccountNumberForChainId(state, highestAccountNumberForChainIdFilter),
-  )
```

Comment:
Nice catch, obviously on first load without any accounts for said chain yet, the highest account number for that chain would be 0, since there are no accounts